src/labels/CustomLabels.labels

```diff
@@ -1564,7 +1668,15 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>RD2_EnablementMigrationCompleteMessage</shortDescription>
-        <value>&lt;b&gt;&lt;i&gt;All done! You’re now running the Enhanced Recurring Donations experience!&lt;/i&gt;&lt;/b&gt;</value>
+        <value>Your migration was successful with no errors, you’re all ready to go with Enhanced Recurring Donations.</value>
+    </labels>
+    <labels>
+        <fullName>RD2_EnablementMigrationErrorMessage</fullName>
```

Comment:
@sam-knox , can you please review data migration complete message displayed when data migration completes successfully and the data migration error message displayed when data migration completes with errors?

Reply:
@RadSFDeveloper How about this:

"Your migration was successful with no errors. All set up steps are now complete. You're ready to begin using Enhanced Recurring Donations."

Reply:
It has been done.